Code/V2/src/themes/index.js

```diff
@@ -1,8 +1,8 @@
 /* 
```

Comment:
Since you already have a merge request with these changes, you should to the following:

1. Remove this file from this commit
2. Merge the other MR (which I have approved)
3. Rebase this branch onto the latest of master